.github/workflows/static.yml

```diff
@@ -50,9 +50,11 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Download dependencies
-        run: composer update --no-interaction --prefer-dist
+        run: |
+          composer require --dev --no-updatepsalm/phar:3.12.1
```

Comment:
I don't think we need this line if it's going to be in the committed composer.json. Also, if it is, why not put phpstan in there too?

Reply:
Oh, sorry. I was just testing locally. It should not be in composer.json. 

Thank you